submissions/kyle-earnest/scripts/article.js

```diff
@@ -44,6 +44,19 @@ Article.loadAll = function(dataWePassIn) {
  source, process it, then hand off control to the View. */
 Article.fetchAll = function() {
   if (localStorage.hackerIpsum) {
+    $.ajax({
+      url: 'data/hackerIpsum.json',
```

Comment:
This works, but the AJAX call is pretty heavy for just getting the eTag. The eTag is stored in the headers, is there a way you could only get the headers of the request instead of the full body? (protip: yes)